github/src/main/kotlin/gropius/sync/github/IssueGrabber.kt

```diff
@@ -24,36 +24,30 @@ import java.time.OffsetDateTime
 
 /**
  * Implementation of Grabber to retrieve issues and cache them in the database
+ * @param apolloClient Apollo Client to use
  */
 class IssueGrabber(
+    private val remote: RepoDescription,
     /**
      * Reference for the spring instance of RepositoryInfoRepository
```

Comment:
please be consistent with the way you document params at least in a class

Reply:
97e7d2bad00bc5df055901a6122ebc81ba380105